monai/losses/image_dissimilarity.py

```diff
@@ -267,7 +265,7 @@ def parzen_windowing_b_spline(self, img: torch.Tensor, order) -> Tuple[torch.Ten
         norm_min = torch.div(min, bin_size, rounding_mode="floor") - padding
 
         # assign bin/window index to each voxel
-        window_term = torch.div(img, bin_size, rounding_mode="floor") - norm_min  # B[NDHW]
+        window_term = torch.div(img, bin_size) - norm_min  # B[NDHW]
```

Comment:
bringing this update-to-date -- I removed the 'floor' rounding mode here, otherwise there's no gradient throughout the computation.